src/components/form/form-item.ts

```diff
@@ -0,0 +1,21 @@
+import settings from 'carbon-components/es/globals/js/settings';
+import { customElement, LitElement, html } from 'lit-element';
+import styles from './form-item.scss';
+
+const { prefix } = settings;
+
+/**
+ * Presentational element for form items
+ */
+@customElement(`${prefix}-form-item`)
+export default class BXFormItem extends LitElement {
+  render() {
+    return html`
+      <div class="bx--form-item">
```

Comment:
To reduce the number of DOM nodes, would it make sense to make `:host(#{$prefix}-form-item)` inherit (`@extend`) `.#{$prefix}--form-item`?